docs/differences_to_eth.md

```diff
@@ -68,13 +68,18 @@ displayed in the wallet accurately represents the actual costs, even though we u
 #### Cross-Contract Call Gas Limit
 
 These resources can also be limited when making a cross-contract call. However, Solidity doesn't allow specifying anything other than `gas_limit`
-for a cross-contract call. We take the `gas_limit` the contract supplies and use that as `ref_time_limit`. The other two resources are just uncapped
-in this case. Please note that uncapped means they are still constrained by the transaction-specified limits, so this cannot be used to trick
+for a cross-contract call. Conceptually, the `gas_limit` is most similar to Polkadots `ref_time_limit`. `revive`, our Solidity compiler, **does not supply any imposed `gas_limit`**, for two reasons:
+1. `gas_limit` and `ref_time_limit` are semantically not the same; blindly passing EVM gas as `ref_time_limit` can lead to unexpected behavior.
+2. The other two resources would just be uncapped anyways (since they don't existing in EVM it's unclear what to supply), making it futile to specify limits for any weight dimension. 
```

Comment:
```suggestion
2. The other two resources would just be uncapped anyways (since they don't exist in EVM, it's unclear what to supply), making it futile to specify limits for any weight dimension. 
```